src/type/__tests__/schema-test.js

```diff
@@ -183,24 +191,35 @@ describe('Type System: Schema', () => {
         fields: {},
       });
 
-      const SomeSubtype = new GraphQLObjectType({
-        name: 'SomeSubtype',
+      const SomeImplementingInterface = new GraphQLInterfaceType({
+        name: 'SomeImplementingInterface',
         fields: {},
         interfaces: () => [SomeInterface],
       });
 
+      const SomeImplementingObject = new GraphQLObjectType({
+        name: 'SomeImplementingObject',
+        fields: {},
+        interfaces: () => [SomeInterface, SomeImplementingInterface],
```

Comment:
No need to modify this test since you explicitly listing both interfaces here

Reply:
Hmm, unless I'm missing something, the only real changes here are to the names, which make them more symmetrical and consistent w/ the spec and the language used in the implementations.

Reply:
I updated all the names to match your suggestions... I'm not sure why GitHub's diffs aren't showing these changes. I'm going to force them to refresh.

Reply:
@mike-marcacci Just to make sure we both on the same page: This test testing that all interfaces returned by `interfaces` Thunk included in the resulting schema. Since you all list all interfaces we can't test if schema traverse interfaces or not so add transitive interfaces doesn't test anything and just makes this test bigger.

Reply:
Aah, I see what you mean; point taken. I've reverted this file to match master.